plugin/core/sessions.py

```diff
@@ -1786,6 +1790,14 @@ def _apply_text_edits(
             return
         apply_text_edits(view, edits, required_view_version=view_version)
 
+    def _check_select_sheets(self, sheets: List[sublime.Sheet]) -> None:
+        if len(sheets) > 1 == len(self.window.selected_sheets()):
```

Comment:
oh man. I don't know what this syntax does exactly and don't know how to google it :)
It's probably just me and maybe we've already discussed it before but still.

Reply:
Haha. Okay I'll reconsider tomorrow.

Reply:
https://github.com/sublimelsp/LSP/pull/2311#discussion_r1299435330 :⁠-⁠)

Reply:
thanks. Wasn't sure if it was the same case :)